crates/apub/src/activities/send/post.rs

```diff
@@ -29,52 +26,20 @@ use lemmy_websocket::LemmyContext;
 
 #[async_trait::async_trait(?Send)]
 impl ApubObjectType for Post {
-  /// Send out information about a newly created post, to the followers of the community.
-  async fn send_create(&self, creator: &Person, context: &LemmyContext) -> Result<(), LemmyError> {
-    let page = self.to_apub(context.pool()).await?;
-
-    let community_id = self.community_id;
-    let community = blocking(context.pool(), move |conn| {
-      Community::read(conn, community_id)
-    })
-    .await??;
-
-    let mut create = Create::new(
-      creator.actor_id.to_owned().into_inner(),
-      page.into_any_base()?,
-    );
-    create
-      .set_many_contexts(lemmy_context()?)
-      .set_id(generate_activity_id(CreateType::Create)?)
-      .set_to(public())
-      .set_many_ccs(vec![community.actor_id()]);
-
-    send_to_community(create, creator, &community, None, context).await?;
-    Ok(())
+  async fn send_create(
+    &self,
+    _creator: &Person,
+    _context: &LemmyContext,
+  ) -> Result<(), LemmyError> {
+    unimplemented!()
```

Comment:
I think I see these are unimplemented because of the new struct impls, but why have this impl at all then, for deletes? Should probably  be done in a uniform way for all the sends.

Reply:
This is still used for comment and private message, which I will rewrite later. I want to avoid rewriting everything at once because it gets too huge and complicated.